float32 motor_current
int32 commanded_value
int32 measured_velocity
int32 measured_position
float32 supply_voltage
float32 supply_current
uint8 estop
